_episodes_rmd/01-raster-structure.Rmd

```diff
@@ -41,6 +41,12 @@ knitr_fig_path("01-")
 > You will need the most current version of `R` and, preferably, `RStudio` loaded
 on your computer to complete this tutorial.
 >
+> ### Setup RStudio Project
+>
+> Make sure you have setup a RStudio project for this lesson, as described in the <a href="{{site.baseurl}}/setup/" target="_blank">setup instructions</a>. 
+>
+> If you don't have a RStudio project, you will need to manually set the working directory with `setwd([your-directory-name])` so R can find the data files to load. 
```

Comment:
hey @pmarchand1  this is fantastic! here is on detail that may confuse folks
How about your replace:
`setwd([your-directory-name])`
with:
`setwd(path-to-your-working-directory)`

to be a bit more precise and specific.

2. the url is great. @fmichonneau  do we need to specify a `baseurl` with the new template? in the old NEON template we did but not in the new template.

Reply:
baseurl was defined in the _config.yml file previously